src/main/java/com/mraof/minestuck/computer/editmode/ServerEditHandler.java

```diff
@@ -335,6 +332,10 @@ public static void onTossEvent(ItemTossEvent event)
 				}
 				else event.setCanceled(true);
 			}
+			else if(AlchemyHelper.isPunchedCard(stack) && DeployList.getEntryForItem(AlchemyHelper.getDecodedItem(stack), data.connection, event.getEntity().level) != null)
+			{
+				event.setCanceled(false);
+			}
```

Comment:
What is the purpose of this new block?

Reply:
What new block? That piece of code is just checking whether the item is a punched card, and whether the item inside of the punched card is in the deploy list. It's not creating a new block?

Reply:
Ah, I meant "code block". It caught me off-guard before I read about the corresponding feature from your list.

Though having read that, should it then rather be checking specifically for atheneum entries?